libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginUsernamePasswordFragment.java

```diff
@@ -161,6 +162,7 @@ public void onEditorCommit() {
 
         mPasswordInput = rootView.findViewById(R.id.login_password_row);
         mPasswordInput.setText(mInputPassword);
+        mPasswordInput.setHint(getString(R.string.password));
```

Comment:
This is a user-facing change, so I'm hoping we can find another way to approach it. Otherwise, we'll need to bring in some of the original devs for the login flow to discuss.